.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
What's this do?

Reply:
This comes with the `pnpm dlx husky-init && pnpm install` command, it loads the Husky shell helpers.